pkg/strutil/json.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package strutil
+
+import "encoding/json"
+
+func MustString(v any) string {
```

Comment:
bad name. `MustXXX` will be panic if error happened.

Reply:
the name should be `TryGetJsonString`

Reply:
resolved